app/assets/javascripts/discourse/app/routes/review-index.js

```diff
@@ -39,6 +39,10 @@ export default DiscourseRoute.extend({
       sort_order: meta.sort_order,
       additionalFilters: meta.additional_filters || {},
     });
+
+    controller.reviewables.forEach((reviewable) => {
```

Comment:
Suggestion: `controller.reviewables.setEach("stale", false)` is a little simpler.

Reply:
TIL about `setEach`